### PR TITLE
ddl, metrics: remove DDL version in metrics

### PR DESCRIPTION
**Pull request by zimulala**
### What problem does this PR solve? 
Related #7472
When the updating schema is frequent, the DDL version will be very much. But we'd better not overuse labels in prometheus. Refer to https://prometheus.io/docs/practices/instrumentation/#do-not-overuse-labels

### What is changed and how it works?
Remove the label of the version in metrics.

### Check List 

Tests 

 - No code

### CODE REVIEW

File: ddl/syncer.go [hunk 1]
```diff
@@ -325,8 +324,7 @@ func (s *schemaVersionSyncer) OwnerCheckAllVersions(ctx context.Context, latestV
 
 	var err error
 	defer func() {
-		ver := strconv.FormatInt(latestVer, 10)
-		metrics.OwnerHandleSyncerHistogram.WithLabelValues(metrics.OwnerGetGlobalVersion, ver, metrics.RetLabel(err)).Observe(time.Since(startTime).Seconds())
+		metrics.OwnerHandleSyncerHistogram.WithLabelValues(metrics.OwnerGetGlobalVersion, metrics.RetLabel(err)).Observe(time.Since(startTime).Seconds())
```
Comment (crazycs520) [inline ddl/syncer.go]: s/OwnerGetGlobalVersion/OwnerCheckAllVersions ?

**Comment by zimulala** [comment]
PTAL @crazycs520 @ciscoxll @winkyao

**Comment by shenli** [comment]
LGTM

**Pull request merged**